### ISSUE TITLE
The problem of Chinese translation into English

### ISSUE THREAD

**Issue by travel-go** [facebookresearch/fairseq-lua#120]
Hello, author. Very grateful for your help to me before, I recently want to see you this set of parameters used in Chinese-English what will happen, but I found it bleu  is only 23, it is too low, in general, is over 30 . Then I use the parameters of the best parameters of the English-German run, but my Chinese-English sentences only 125w. The parameters are as follows:

 fairseq train -sourcelang ch -targetlang en -datadir data-bin/fconv_zh_en/ -model fconv -nembed 768 -nhid 512 -fconv_nhids 512,512,512,512,512,512,512,512,512,1024,1024,1024,1024,2048,2048 -fconv_nlmhids 512,512,512,512,512,512,512,512,512,1024,1024,1024,1024,2048,2048 -fconv_kwidths 3,3,3,3,3,3,3,3,3,3,3,3,3,1,1 -fconv_klmwidths 3,3,3,3,3,3,3,3,3,3,3,3,3,1,1 -optim nag -timeavg -lr 0.25 -momentum 0.99 -clip 0.1 -dropout 0.1 -maxbatch 1200 -bptt 0 -batchsize 48 -maxepoch 50 -minepochtoanneal 20 -ndatathreads 4 -seed 1 -log 

After that I found him converging too fast, leading to overfitting, I do not know if you have run the Chinese-English data, or Can you give me some advice?

**Comment by travel-go** [comment]
I find it is my problem.Sorry
